src/tests/unit/tests/issue-filing/common/create-file-issue-handler.test.ts

```diff
@@ -1,45 +1,60 @@
+import { IMock, Mock, MockBehavior } from 'typemoq';
+import { Tabs } from 'webextension-polyfill-ts';
 
 describe('createFileIssueHandler', () => {
-    it('properly files an issue', () => {
-        const serviceMap: IssueFilingServicePropertiesMap = {};
-        const issueData: CreateIssueDetailsTextData = {
-            targetApp: {
-                name: 'pageTitle<x>',
-            },
-        } as any;
-        const environmentInfoStub: EnvironmentInfo = {
-            axeCoreVersion: 'test axe version',
-            browserSpec: 'test browser spec',
-            extensionVersion: 'test extension version',
-        };
-        const settingsStub = {
-            repo: 'test-repo',
-        };
-        const urlStub = 'url-stub';
-
-        const settingsGetterMock = Mock.ofType<(data: IssueFilingServicePropertiesMap) => any>(undefined, MockBehavior.Strict);
+    const serviceMap: IssueFilingServicePropertiesMap = {};
+    const issueData: CreateIssueDetailsTextData = {
+        targetApp: {
+            name: 'pageTitle<x>',
+        },
+    } as CreateIssueDetailsTextData;
+    const environmentInfoStub: EnvironmentInfo = {
+        axeCoreVersion: 'test axe version',
+        browserSpec: 'test browser spec',
+        extensionVersion: 'test extension version',
+    };
+    const settingsStub = {
+        repo: 'test-repo',
+    };
+    const urlStub = 'url-stub';
+
+    let settingsGetterMock: IMock<(data: IssueFilingServicePropertiesMap) => any>;
+    let urlProviderMock: IMock<IssueFilingUrlProvider<any>>;
+    let browserAdapterMock: IMock<BrowserAdapter>;
+
+    beforeEach(() => {
+        settingsGetterMock = Mock.ofType<(data: IssueFilingServicePropertiesMap) => any>(undefined, MockBehavior.Strict);
         settingsGetterMock.setup(getter => getter(serviceMap)).returns(() => settingsStub);
 
-        const urlProviderMock = Mock.ofType<IssueFilingUrlProvider<any>>(undefined, MockBehavior.Strict);
+        urlProviderMock = Mock.ofType<IssueFilingUrlProvider<any>>(undefined, MockBehavior.Strict);
         urlProviderMock.setup(provider => provider(settingsStub, issueData, environmentInfoStub)).returns(() => urlStub);
 
-        const browserAdapterMock = Mock.ofType<BrowserAdapter>(undefined, MockBehavior.Strict);
-        browserAdapterMock.setup(adapter => adapter.createTab(urlStub)).verifiable(Times.once());
+        browserAdapterMock = Mock.ofType<BrowserAdapter>(undefined, MockBehavior.Strict);
+    });
+
+    it('properly files an issue', async () => {
+        browserAdapterMock.setup(adapter => adapter.createActiveTab(urlStub)).returns(() => Promise.resolve({} as Tabs.Tab));
 
         const testSubject = createFileIssueHandler(urlProviderMock.object, settingsGetterMock.object);
 
-        testSubject(browserAdapterMock.object, serviceMap, issueData, environmentInfoStub);
+        await expect(testSubject(browserAdapterMock.object, serviceMap, issueData, environmentInfoStub)).resolves.toBe(undefined);
```

Comment:
Should this be verifying that the correct side effects happen?

Reply:
good catch.